### ISSUE TITLE
For Navsim training and evaluation, can you provide a visualization tool for model inference results?

### ISSUE THREAD

**Issue by chajchaj** [Kguo-cs/iPad#15]
Thanks!

**Comment by gzspkgypt** [comment]
<img width="1800" height="774" alt="Image" src="https://github.com/user-attachments/assets/f58352a3-fd72-40d8-9fc4-d49d4e0e269f" />
We have already implemented visualization based on Navsim, which will be open-sourced along with the training code.

**Comment by qwefvb** [comment]
@gzspkgypt 

During the training process, the log keeps printing the following information, and the training proceeds extremely slowly. Do you have any solutions?

/utils/tensor_numpy.cpp:206.)

mask = torch.from_numpy(mask).to(x.dtype).to(x.device)

Ray objects: 0%| | 0/16 [00:00<?, ?it/s]

(wrapped_fn pid=3697985) /mnt/ssd/home/liyucheng/miniconda3/envs/ipad/lib/python3.8/site-packages/shapely/linear.py:88: RuntimeWarning: invalid value encountered in line_locate_point

(wrapped_fn pid=3697985) return lib.line_locate_point(line, other)

Ray objects: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 16/16 [00:10<00:00, 1.50it/s]

**Comment by gzspkgypt** [comment]
@qwefvb Directly use the control platform, the script does not meet the warning message, and the warning message is weak.
